middleman-search.gemspec

```diff
@@ -18,9 +18,9 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "middleman-core", ["~> 3.2"]
-  spec.add_dependency "therubyracer", ["~> 0.12"]
-  spec.add_dependency "nokogiri", ["~> 1.5"]
+  spec.add_dependency "middleman-core", ["~> 4.0"]
```

Comment:
Thanks for the PR! We'd like to keep support for 3.x, and this change would force users to work with 4.0. Do you think you could do something like https://github.com/middleman/middleman-livereload/commit/4f619cf5aa0f8fc2f307196f79a4ffeefd04bae9#diff-4d67daa0283bf974174fb193b9560c18 to accept both majors?